marko/ext/latex_renderer.py

```diff
@@ -0,0 +1,168 @@
+"""
+LaTeX renderer
+"""
+import logging
+from typing import Iterable, List, Set, Union
+
+from marko import Renderer
+
+_logger = logging.getLogger(__name__)
+
+
+class LatexRenderer(Renderer):
+    """Render the parsed Markdown to LaTeX format."""
+
+    packages: Set[str]
+    _packages_back: Set[str]
+
+    # Header levels that will be numbered. Default none.
+    # It could be a list of header levels (ex. [1, 2]) or header names ['part', 'section']
+    # TODO: add unit test
+    numbered_headers: List[Union[int, str]]
+
+    def __init__(self,
+                 initial_packages: Iterable[str] = (),
```

Comment:
Is there any chance we can pass in the initial parameters?

Reply:
* "initial_packages": we can skip them.
* "numbered_headers": I didn't need them in my use case, just make all unnumbered, but I believe others might need to make some header numbered.

Changing numbered_header can be don't also by subclassing the renderer or use mixin. I prefer configuring the renderer through changing some parameter. What do you think?

Reply:
Since we currently don't have such a mechanism to pass in arguments to the renderer class, can we remove them from the arguments?

Let's see how to make it possible in another enhancement.

Reply:
Sure